lib/oci_migrate/migrate/doc/oci_image_migrate.md

```diff
@@ -0,0 +1,180 @@
+# oci-utils.oci-image-migrate
```

Comment:
This seems more of a developer type doc than a usage/overview one, maybe split this up moving the developer specific content into a developer file